dts/bindings/led_strip/worldsemi,ws2812-gpio.yaml

```diff
@@ -6,9 +6,54 @@ description: |
 
   Driver bindings for bit-banging a WS2812 or compatible LED strip.
 
-  The GPIO driver uses inline assembly, and isn't available for all
-  boards.
+  The GPIO driver uses inline assembly, and isn't available for all boards. The
+  timing is automatically derived from the cpu clock frequency, or can be
+  provided by setting the delay_txx properties in the device tree, or can be
+  manually provided by the Kconfig settings DELAY_Txx.
+
+  / {
+        cpus {
+            cpu@0 {
+                clock-frequency = <64000000>;
+            };
+        };
+
+        chosen {
+            zephyr,led-strip = &rgb_led;
+        };
+
+        rgb_led: ws2812 {
+            compatible = "worldsemi,ws2812-gpio";
+            chain-length = <1>;
+            color-mapping = <LED_COLOR_ID_GREEN
+                             LED_COLOR_ID_RED
+                             LED_COLOR_ID_BLUE>;
+            reset-delay = <50>;
+            gpios = <&gpio0 16 0>;
+            delay_t1h = <48>;
+            delay_t1l = <32>;
+            delay_t0h = <16>;
+            delay_t0l = <32>;
+        };
+  };
 
 compatible: "worldsemi,ws2812-gpio"
 
 include: [base.yaml, ws2812-gpio.yaml]
+
+properties:
+  delay_t1h:
+    type: int
+    description: Number of NOP assembly operations to create a 700 nsec delay for an 1 bit (high pulse)
```

Comment:
Have you sufficiently explained the meanings of T0L, T0H, T1L, and T1H here? You can refer to the datasheet, but please clarify which part of the waveform is specified.
Also, since using NOP explains the implementation method, it's no problem to mention it here, but it should remain a secondary explanation.

I wrote below how it is in the rpi_pico_pio case.

https://github.com/zephyrproject-rtos/zephyr/blob/b9a4900c3002b50c1475da2b05457161ad83dc9a/dts/bindings/led_strip/worldsemi%2Cws2812-rpi_pico-pio.yaml#L11-L43

Reply:
I've updated the description and used your example, let me know if this is good enough.